modules/apps/transfer/keeper/migrations.go

```diff
@@ -53,6 +54,34 @@ func (m Migrator) MigrateTraces(ctx sdk.Context) error {
 	return nil
 }
 
+// MigrateTotalEscrowOut migrates the total escrow amount to calculate total IBC'd out.
+func (m Migrator) MigrateTotalEscrowOut(ctx sdk.Context) error {
+	getExistingChannels := m.keeper.channelKeeper.GetAllChannelsWithPortPrefix(ctx, types.PortID)
+
+	var nativeDenom string
+	escrowAmount := sdk.ZeroInt()
+
+	for _, channel := range getExistingChannels {
+		escrowAddress := types.GetEscrowAddress(types.PortID, channel.ChannelId)
+		getEscrowBalances := m.keeper.bankKeeper.GetAllBalances(ctx, escrowAddress)
+
+		for _, escrowBalance := range getEscrowBalances {
```

Comment:
Note: very hacking way of finding sourceChain native denom. Please let me know if there is a better way of retrieving it :))